support/include/sleigh/libsleigh.hh

```diff
@@ -15,117 +15,117 @@
+#include <ghidra/grammar.hh>
+#include <ghidra/graph.hh>
+#include <ghidra/heritage.hh>
+#include <ghidra/ifacedecomp.hh>
+#include <ghidra/ifaceterm.hh>
+#include <ghidra/inject_ghidra.hh>
+#include <ghidra/inject_ghidra.hh>
+#include <ghidra/interface.hh>
+#include <ghidra/jumptable.hh>
+#include <ghidra/libdecomp.hh>
+#include <ghidra/loadimage.hh>
+#include <ghidra/loadimage_ghidra.hh>
+#include <ghidra/loadimage_xml.hh>
+#include <ghidra/memstate.hh>
+#include <ghidra/merge.hh>
+#include <ghidra/op.hh>
+#include <ghidra/opbehavior.hh>
+#include <ghidra/opcodes.hh>
+#include <ghidra/options.hh>
+#include <ghidra/override.hh>
+#include <ghidra/paramid.hh>
+#include <ghidra/partmap.hh>
+#include <ghidra/pcodecompile.hh>
+#include <ghidra/pcodeinject.hh>
+#include <ghidra/pcodeparse.hh>
+#include <ghidra/pcoderaw.hh>
+#include <ghidra/prefersplit.hh>
+#include <ghidra/prettyprint.hh>
+#include <ghidra/printc.hh>
+#include <ghidra/printjava.hh>
+#include <ghidra/printlanguage.hh>
+#include <ghidra/rangemap.hh>
+#include <ghidra/rangeutil.hh>
+#include <ghidra/raw_arch.hh>
+#include <ghidra/ruleaction.hh>
+#include <ghidra/rulecompile.hh>
+#include <ghidra/semantics.hh>
+#include <ghidra/sleigh.hh>
+#include <ghidra/ghidra_arch.hh>
+#include <ghidra/sleighbase.hh>
+#include <ghidra/slgh_compile.hh>
 // This is required because slghparse.hh does not have a namespace block
 namespace ghidra {
-#include <sleigh/slghparse.hh>
+#include <ghidra/slghparse.hh>
 } // End namespace ghidra
-#include <sleigh/slghpatexpress.hh>
-#include <sleigh/slghpattern.hh>
-#include <sleigh/slghsymbol.hh>
-#include <sleigh/space.hh>
-#include <sleigh/string_ghidra.hh>
-#include <sleigh/stringmanage.hh>
-#include <sleigh/subflow.hh>
-#include <sleigh/testfunction.hh>
-#include <sleigh/transform.hh>
-#include <sleigh/translate.hh>
-#include <sleigh/type.hh>
-#include <sleigh/typegrp_ghidra.hh>
-#include <sleigh/typeop.hh>
-#include <sleigh/types.h>
-#include <sleigh/unify.hh>
-#include <sleigh/userop.hh>
-#include <sleigh/variable.hh>
-#include <sleigh/varmap.hh>
-#include <sleigh/varnode.hh>
-#include <sleigh/xml.hh>
-#include <sleigh/xml_arch.hh>
-#include <sleigh/unionresolve.hh>
-#include <sleigh/marshal.hh>
+#include <ghidra/slghpatexpress.hh>
+#include <ghidra/slghpattern.hh>
+#include <ghidra/slghsymbol.hh>
+#include <ghidra/space.hh>
+#include <ghidra/string_ghidra.hh>
+#include <ghidra/stringmanage.hh>
+#include <ghidra/subflow.hh>
+#include <ghidra/testfunction.hh>
+#include <ghidra/transform.hh>
+#include <ghidra/translate.hh>
+#include <ghidra/type.hh>
+#include <ghidra/typegrp_ghidra.hh>
+#include <ghidra/typeop.hh>
+#include <ghidra/types.h>
+#include <ghidra/unify.hh>
+#include <ghidra/userop.hh>
+#include <ghidra/variable.hh>
+#include <ghidra/varmap.hh>
+#include <ghidra/varnode.hh>
+#include <ghidra/xml.hh>
+#include <ghidra/xml_arch.hh>
+#include <ghidra/unionresolve.hh>
+#include <ghidra/marshal.hh>
 
 #ifdef sleigh_RELEASE_IS_HEAD
```

Comment:
I just reverted the change here since I don't think we want to change the name of the CMake variable. Otherwise, this is looking good and I agree with the general idea.

Reply:
🤦 Obviously I didn't review this too thoroughly before opening the PR. My bad.

Thank you for the fix!